orders.html

```diff
@@ -90,14 +90,69 @@ <h1>Orders</h1>
 
 										<label>
 											When do you need the cake by?
-											<input type="date" name="date">
+											<input type="date" name="date" id="orderNeededDate" required>
 										</label>
 
 										<label>
 											What kind of event are you celebrating? <i>(eg: birthday, anniversary, etc)</i>
 											<input type="text" name="event">
 										</label>
 
+										<label class="cakeFlavors">
+											Cake Flavor
+											<select name="cake flavor" required>
+												<option value="">- Choose one -</option>
+												<option value="1">Vanilla</option>
+												<option value="2">Chocolate</option>
+											</select>
+										</label>
+
+										<label class="frostingFlavors">
+											Frosting Flavor
+											<select name="frosting flavor" required>
+												<option value="">- Choose one -</option>
+												<option value="1">Vanilla</option>
+												<option value="2">Chocolate</option>
+											</select>
+										</label>
+
+										<label class="dietRestrictions">
+											Please list any allergies or dietary restrictions
+											<input type="text" name="dietRestrictions" placeholder="eggs, soy, wheat, etc. (leave blank if n/a)">
+										</label>
+
+										<label class="colorPalette">
```

Comment:
this is my first take on the color palette. it is optional but i wonder if the average user could handle this.

other thoughts:  link to their party theme/inspiration

Reply:
i like the color palette but i wonder if that will overwhelm average users. Let's wait for Amanda to respond to my email and we can ask her about this feature.